# INSTALL_LuMP_Pulsar_PSRCAT_P0.patch
# patch file 0 for PSRCAT
# 2014 Oct 07  James Anderson  --- GFZ start
#
# Copyright (c) 2014 James M. Anderson <anderson@gfz-potsdam.de>
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
#
#
# Note that the PSRCAT software in this patch is presumably covered by a different
# copyright.  See http://www.atnf.csiro.au/people/pulsar/psrcat/
# for more information about PSRCAT copyright issues.
#
################################################################################
diff -Naur psrcat_tar/makeit psrcat_tar_new/makeit
--- psrcat_tar/makeit	2014-05-19 06:48:53.000000000 +0200
+++ psrcat_tar_new/makeit	2014-10-07 07:34:15.271168511 +0200
@@ -2,18 +2,18 @@
 #
 echo "Clean-up"
 
-\rm *.o
-\rm libpsrcat.a
+#\rm *.o
+#\rm libpsrcat.a
 
 echo "Compiling ..."
 
-gcc -c defineParams.c evalkern.c externalCall.c galcoord_equatorial.c plotParams.c readCatalogue.c displayOutput.c evalwrap.c galcoord_ecliptic.c parseParameters.c wc_strncmp.c
+gcc ${MY_EXTRA_CFLAGS} -fPIC -c defineParams.c evalkern.c externalCall.c galcoord_equatorial.c plotParams.c readCatalogue.c displayOutput.c evalwrap.c galcoord_ecliptic.c parseParameters.c wc_strncmp.c
 
 ar crv libpsrcat.a defineParams.o evalkern.o externalCall.o galcoord_equatorial.o plotParams.o readCatalogue.o displayOutput.o evalwrap.o galcoord_ecliptic.o parseParameters.o wc_strncmp.o
 
 ranlib libpsrcat.a
 
-gcc -o psrcat psrcat.c -L./ -lpsrcat -lm 
+gcc ${MY_EXTRA_CFLAGS} -fPIC  -o psrcat psrcat.c -L./ -lpsrcat -lm 
 
 echo "Complete"
 #set dir = `pwd`
